tokio/src/runtime/metrics/worker.rs

```diff
@@ -57,32 +70,40 @@ pub(crate) struct WorkerMetrics {
 }
 
 impl WorkerMetrics {
-    pub(crate) fn from_config(config: &Config) -> WorkerMetrics {
-        let mut worker_metrics = WorkerMetrics::new();
-        worker_metrics.poll_count_histogram = config
-            .metrics_poll_count_histogram
-            .as_ref()
-            .map(|histogram_builder| histogram_builder.build());
-        worker_metrics
+    cfg_not_unstable_metrics! {
```

Comment:
Let's move this down to be right above the `cfg_unstable_metrics!` block so that we keep the conditionally compiled implementations together.